e2e_playwright/conftest.py

```diff
@@ -831,22 +831,13 @@ def wait_for_app_run(
         page.wait_for_timeout(wait_delay)
 
 
-def wait_for_app_loaded(
-    page: Page,
-    embedded: bool = False,
-):
+def wait_for_app_loaded(page: Page):
     """Wait for the app to fully load."""
     # Wait for the app view container to appear:
     page.wait_for_selector(
         "[data-testid='stAppViewContainer']", timeout=30000, state="attached"
     )
 
-    # Wait for the main menu to appear:
-    if not embedded:
-        page.wait_for_selector(
-            "[data-testid='stMainMenu']", timeout=20000, state="attached"
-        )
```

Comment:
I removed this since this additional check isn't really needed. The check above + the checks in `wait_for_app_run` should be enough to determine if the app is ready. And we want to start testing apps that don't have a main menu.